test/LaunchDarkly.EventSource.Tests/EventSourceTests.cs

```diff
@@ -356,20 +371,91 @@ public void Exponential_backoff_should_not_exceed_maximum_in_test_loop()
 
             handler.QueueResponse(new HttpResponseMessage(HttpStatusCode.NoContent));
             
-            var eventSource = new EventSource(new Configuration(_uri, handler, closeOnEndOfStream: true));
+            var evt = new EventSource(new Configuration(_uri, handler, readTimeout:_defaultReadTimeout));
 
             var backoffs = new List<TimeSpan>();
-            eventSource.Error += (_, e) =>
+            evt.Error += (_, e) =>
             {
-                backoffs.Add(eventSource.BackOffDelay);
+                backoffs.Add(evt.BackOffDelay);
             };
 
             //Act
-            await eventSource.StartAsync();
+            await evt.StartAsync();
 
             //// Assert
             Assert.NotEmpty(backoffs);
             Assert.True(backoffs.Distinct().Count() == backoffs.Count());
         }
+
+
+        [Fact]
+        public async Task When_response_exceeds_read_timeout_then_read_timeout_exception_occurs()
+        {
+            // Arrange
```

Comment:
You could add a bit more to the comment or just remove it. ;)

Reply:
I'll remove the comment.